partner_email_check/models/res_partner.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+import logging
+from odoo import api, models, _
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from validate_email import validate_email
+except ImportError:
+    _logger.debug("Cannot import `validate_email`.")
```

Comment:
I think we should log an error and use double quotes instead of `.

_logger.error('Cannot import "validate_email".')